prototypes/recipe-circuit.lua

```diff
@@ -0,0 +1,51 @@
+data:extend(
+{
+  {
+    type = "recipe",
+    name = "copper-cable",
+
+    energy_required = 2,
+    ingredients = {{"copper-plate", 10}},
+    result = "copper-cable",
+    result_count = 2,
+  },
+  {
+    type = "recipe",
+    name = "electronic-circuit",
+
+    energy_required = 8,
+    ingredients = {
+      {"iron-plate", 10},
+      {"copper-cable", 10},
+    },
+    result = "electronic-circuit",
+  },
+  --[[
+  {
+    type = "recipe",
+    name = "advanced-circuit",
+    enabled = false,
+
+    energy_required = 16,
+    ingredients = {
+      {"electronic-circuit", 5},
+      {"plastic-bar", 5},
+      {"copper-cable", 10},
+    },
+    result = "advanced-circuit",
+  },
+  ]]
```

Comment:
The recipe for the advanced circuit was updated twice:

https://github.com/Afforess/FactorioMod-Marathon/blob/0.5.4/prototypes/recipe.lua#L151
and
https://github.com/Afforess/FactorioMod-Marathon/blob/0.5.4/prototypes/recipe.lua#L732

The latter one overwrites the first one, so I copied both and commented the first one out.